spec.html

```diff
@@ -23761,6 +24667,13 @@ <h1>EvalDeclarationInstantiation ( _body_, _varEnv_, _lexEnv_, _strict_ )</h1>
                     1. NOTE: Annex <emu-xref href="#sec-variablestatements-in-catch-blocks"></emu-xref> defines alternate semantics for the above step.
                   1. NOTE: A direct eval will not hoist var declaration over a like-named lexical declaration.
               1. Set _thisEnv_ to _thisEnv_.[[OuterEnv]].
+          1. Let _privateIdentifiers_ be a new empty List.
+          1. Let _privEnv_ be _privateEnv_.
+          1. Repeat, while _privEnv_ is not *null*,
+            1. For each binding named _N_ in _privEnv_, do
```

Comment:
I don't love this wording, but I don't have a better suggestion at the moment. I look forward to improving it in https://github.com/tc39/ecma262/pull/2288.

Reply:
No longer relevant as of https://github.com/tc39/ecma262/pull/1668/commits/fa2a42430b32a0f074906f68d6d078fd09d8d8cc.